heffalump/heffalump.go

```diff
@@ -50,8 +58,24 @@ func (h *Heffalump) WriteHell(bw *bufio.Writer) (int64, error) {
 	buf := h.pool.Get().([]byte)
 	defer h.pool.Put(buf)
 
-	if _, err = bw.WriteString("<html>\n<body>\n"); err != nil {
-		return n, err
+	switch cType {
+	case PlainText:
+		if _, err = bw.WriteString("# Chapter 1\n"); err != nil {
+			return n, err
+		}
+		break
+	case HTML:
+		if _, err = bw.WriteString("<html>\n<body>\n"); err != nil {
+			return n, err
+		}
+		break
+	case JSON:
+		if _, err = bw.WriteString("[\""); err != nil {
```

Comment:
also you're discarding the bytes written (`n`) but then returning the variable still

edit: looks like i was too, woops :^)
still probably worth cleaning up since you're doing surgery over here, otherwise I will after the fact